recipes/zocalo-dls/meta.yaml

```
@@ -0,0 +1,49 @@
+{% set name = "zocalo-dls" %}
+{% set version = "0.2.0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/zocalo_dls-{{ version }}.tar.gz"
+  sha256: "c31dccec0732985d81181629a49b0afb025c55cf0c0eb1aadaaa15bc83f0f592"
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
```

No need for `--no-deps --ignore-installed` anymore.
```suggestion
  script: "{{ PYTHON }} -m pip install . -vv "
```